Source/loadsave.cpp

```diff
@@ -1528,6 +1530,45 @@ void SaveDroppedItemLocations(SaveHelper &file, const std::unordered_map<uint8_t
 	}
 }
 
+constexpr uint32_t VersionAdditionalMissiles = 0;
+
+void SaveAdditionalMissiles(int missileCountAdditional)
+{
+	SaveHelper file("additionalMissiles", sizeof(uint32_t) + sizeof(int32_t) + (missileCountAdditional * 180));

```

Comment:
180?

Reply:
`SaveMissile` writes 180 bytes 🙂

Reply:
Could you add it as a constant so that we know what it is :)

Reply:
Done. 😉